src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/ASEReadCounter.java

```diff
@@ -269,30 +270,18 @@ else if (base.getBase() == altAllele) {
         }
 
         final StringBuilder line = new StringBuilder();
-        line.append(pileup.getLocation().getContig());
-        line.append(separator);
-        line.append(pileup.getLocation().getStart());
-        line.append(separator);
-        line.append(siteID);
-        line.append(separator);
-        line.append(refAllele);
-        line.append(separator);
-        line.append(altAllele);
-        line.append(separator);
-        line.append(refCount);
-        line.append(separator);
-        line.append(altCount);
-        line.append(separator);
-        line.append(totalNonFilteredCount);
-        line.append(separator);
-        line.append(lowMAPQDepth);
-        line.append(separator);
-        line.append(lowBaseQDepth);
-        line.append(separator);
-        line.append(rawDepth);
-        line.append(separator);
-        line.append(otherBasesCount);
-        line.append(separator);
+        line.append(pileup.getLocation().getContig()).append(separator);
```

Comment:
I don't think the StringBuilder makes it that messy, so I'm going to leave it as is for now unless you have a strong opinion.